### PR TITLE
Setup prefer-object-spread eslint rule - Closes #1966

### LINKED ISSUE #1966: Setup eslint prefer-object-spread rule
The goal of this ticket is to set up eslint [prefer-object-spread](https://eslint.org/docs/rules/prefer-object-spread) rule and update the codebase to satisfy the rule.

There are currently 65 uses of `Object.assign`:
```
$ git grep 'Object.assign' | wc -l
      65
```

### PR DESCRIPTION
**Pull request by slaweet**
### What issue have I solved?

#1966

### How have I implemented/fixed it?

Edit `.eslintrc` and run  `npm run eslint -- --fix`

### How has this been tested?

`npm run eslint`

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/actions/account.js [hunk 1]
```diff
@@ -132,10 +132,10 @@ export const updateDelegateAccount = ({ publicKey }) =>
     const liskAPIClient = getState().peers.liskAPIClient;
     return getDelegate(liskAPIClient, { publicKey })
       .then((response) => {
-        dispatch(accountUpdated(Object.assign(
-          {},
-          { delegate: response.data[0], isDelegate: true },
-        )));
+        dispatch(accountUpdated({
+
+          delegate: response.data[0], isDelegate: true,
```
Comment (osvaldovega) [inline src/actions/account.js]: I think here you missed an extra empty line

File: src/components/hwWallet/ledgerLogin.js [hunk 2]
```diff
@@ -89,11 +89,11 @@ class LedgerLogin extends React.Component {
...
 
   changeAccountNameInput(value = '', account) {
-    const newHardwareAccountsName = Object.assign(
-      {},
-      this.state.hardwareAccountsName,
-      { [account]: value.length < 20 ? value : value.substr(0, 20) },
-    );
+    const newHardwareAccountsName = {
+
+      ...this.state.hardwareAccountsName,
```
Comment (osvaldovega) [inline src/components/hwWallet/ledgerLogin.js]: here too, extra empty line

File: src/actions/account.js (hunk 1 above)
Comment (slaweet) [reply src/actions/account.js]: Fixed

File: src/components/hwWallet/ledgerLogin.js (hunk 2 above)
Comment (slaweet) [reply src/components/hwWallet/ledgerLogin.js]: Fixed

**Pull request merged**